email_automation/welcome-template.txt

```diff
@@ -6,8 +6,8 @@
 <br>
  <strong>Your admin dashboard can be found at:</strong> <br>
  <br>
-    <a href='https://<pool_name>-openpath.nrel.gov/admin/'>
-    https://<pool_name>-openpath.nrel.gov/admin/</a>.<br>
+    <a href='https://nrelopenpath-prod-<pool_name>-openpath.nrel.gov/admin/'>
+    https://nrelopenpath-prod-<pool_name>-openpath.nrel.gov/admin/</a>.<br>
```

Comment:
I think this was supposed to fix https://github.com/e-mission/e-mission-docs/issues/1008#issuecomment-1839210067 but it is off. (sorry, partially my bad - I typed the original comment out in a hurry and edited it later).
This makes the URL even more wrong.

You should definitely test with other configs though and make sure that the link is valid.

Reply:
>You should definitely test with other configs though and make sure that the link is valid.

Are there any specific ones that you're okay with me adding myself to? I don't want to bother the program admins if I can avoid it, and I'm not sure if they get any emails when someone is added to their pools.

Reply:
> Make sure that you can actually login using the link in the email!

I've tested many times with both --local and --github, and realized that the variable `welcome` needed to be the same for both local and GitHub runs to be able to access welcome-template.txt. I've pushed that change and it's now ready!